llvm/tools/llvm-objdump/SourcePrinter.h

```diff
@@ -166,6 +169,114 @@ class SourcePrinter {
                                StringRef Delimiter = "; ");
 };
 
+enum VisualizeJumpsMode : int {
+  Off = 0,
+
+  ColorAuto = 0x1,
+  Color3Bit,
+  //Color8Bit, // TODO GNU objdump can use more colors in 256-color terminals
+
+  CharsASCII = 0x10,
+  CharsUnicode = 0x20,
+
+  ColorMask = 0xf,
+  CharsMask = 0xf0,
+};
+
```

Comment:
Nit: double blank line. Is this file clang-formatted?

Reply:
Clang-formatted the whole patch set.